### ISSUE TITLE
[feature request] Pick up all slots when dragging the cursor really fast

### ISSUE THREAD

**Issue by Siphalor** [Siphalor/mouse-wheelie#135]
**Is your feature request related to a problem? Please describe.**
Currently, if the cursor moves to fast, the game/mod skips certain slots in the middle of the movement.

**Describe the solution you'd like**
The mod could look at the latest two mouse move events and draw a virtual line between them to find additional slots.
This feature should probably be configurable as an opt-out, as there might be some smaller issues with this feature.

**Describe alternatives you've considered**
Basically not doing this since this is mostly a quality of life thing.

**Additional context**
Originally suggested in #133.
